Source/WebCore/Modules/webxr/WebXRSystem.cpp

```diff
@@ -275,6 +275,9 @@ bool WebXRSystem::isFeaturePermitted(PlatformXR::SessionFeature feature) const
     case PlatformXR::SessionFeature::ReferenceSpaceTypeViewer:
         return true;
     case PlatformXR::SessionFeature::WebGPU:
+#if ENABLE(WEBXR_HIT_TEST)
+    case PlatformXR::SessionFeature::HitTest:
+#endif
```

Comment:
Hit testing requires xr-spatial-tracking so we need to check if the feature is enabled in the permissions policy

Reply:
Fixed.